dotCMS/src/integration-test/java/com/dotcms/publishing/job/TestCaseSiteSearchBuilder.java

```diff
@@ -0,0 +1,85 @@
+package com.dotcms.publishing.job;
+
+import com.dotcms.publishing.job.SiteSearchJobImplTest.ExpectedResults;
+import com.dotcms.publishing.job.SiteSearchJobImplTest.TestCaseSiteSearch;
+
+public class TestCaseSiteSearchBuilder {
+
+    private boolean defaultPageToDefaultLanguage;
+    private boolean defaultContentToDefaultLanguage;
+    private boolean siteSearchDefaultLanguage;
+    private boolean siteSearchSecondLanguage;
+    private boolean createContentInDefaultLanguage;
+    private boolean createContentInSecondLanguage;
+    private boolean createPageInDefaultLanguage;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field createPageInDefaultLanguage has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)